packages/glibc.rb

```diff
@@ -167,6 +167,9 @@ def self.install
       system "localedef --prefix=#{CREW_DEST_DIR} -i tr_TR -f UTF-8 tr_TR.UTF-8"
       system "localedef --prefix=#{CREW_DEST_DIR} -i zh_CN -f GB18030 zh_CN.GB18030"
     end
+
+    FileUtils.mkdir "#{CREW_DEST_PREFIX}/include/malloc"
+    FileUtils.symlink "../malloc.h", "#{CREW_DEST_PREFIX}/include/malloc/malloc.h"
```

Comment:
This would require new binaries and bumping the version.

Reply:
Okay we can remove it. It's not acutely necessary by any means.